src/components/pages/order/navbar/NavbarRightSide.js

```diff
@@ -1,45 +1,45 @@
-import React from "react";
+import { useState } from "react";
+import { toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 import styled from "styled-components";
-import { theme } from "../../../../theme";
+import AdminToast from "./AdminToast";
 import Profile from "./Profile";
+import ToggleButton from "../../../reusable-ui/ToggleButton";
 
 export default function NavbarRightSide() {
+  const [adminMode, setAdminMode] = useState(false);
```

Comment:
```suggestion
  const [isAdminMode, setIsAdminMode] = useState(false);
```

Règle de nommage Javascript, pour les booléan, on va préférér un nom qui commence par isQqch ou hasQqch ou canQqch comme si on posait une question.